docs/develop/add-in-manifests.md

```diff
@@ -487,10 +487,14 @@ For troubleshooting issues with your manifest, see [Validate and troubleshoot is
 
 ## See also
 
+* [How to find the proper order of manifest elements](manifest-element-ordering.md)
 * [Create add-in commands in your manifest][add-in commands]
 * [Specify Office hosts and API requirements](specify-office-hosts-and-api-requirements.md)
 * [Localization for Office Add-ins](localization.md)
 * [Schema reference for Office Add-ins manifests](https://github.com/OfficeDev/office-js-docs-pr/tree/master/docs/overview/schemas)
+* [Update API and manifest version](update-your-javascript-api-for-office-and-manifest-schema-version.md)
+* [Identify an equivalent COM add-in](make-office-add-in-compatible-with-existing-com-add-in.md)
+* [Requesting permissions for API use in content and task pane add-ins](requesting-permissions-for-api-use-in-content-and-task-pane-add-ins.md)
```

Comment:
Another off-topic observation: this title specifies "content and task pane add-ins". Is this as opposed to Custom Function add-ins? Could this article's title just be "Requesting permissions for API use in add-ins" and be just as informative?